benchmarks/k8s/images/driver/Dockerfile

```diff
@@ -12,45 +12,70 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM debian:jessie
+FROM debian:stretch
+
+ARG REPOSITORY=grpc/grpc
+ARG GITREF=master
+
+RUN apt-get update && apt-get install -y git
+
+RUN mkdir -p /grpc/grpc
+WORKDIR /grpc/grpc
+
+RUN git clone https://github.com/$REPOSITORY.git .
+RUN git submodule update --init
+FROM l.gcr.io/google/bazel:0.17.1
```

Comment:
This multi-stage process addresses some feedback on the slow build times experienced by prototype users. It is much faster and appears to condense the size of the image by about 70%.